distribution/debian/postinst

```diff
@@ -10,24 +12,14 @@ if ! getent passwd "$USER" >/dev/null; then
     usermod -a -G "$USER" "$USER"
   fi
 fi
-  
-# Create home directory
 
+# Create home directory
 if [ ! -d /var/opt/sonarr ]; then
   mkdir -p /var/opt/sonarr
-  chown $USER:$USER /var/opt/sonarr
+  chown -R $USER: /var/opt/sonarr
```

Comment:
Not familiar with `$USER:` does that change the user and the group to the same or only the user?

Reply:
"If a colon but no group name follows the user name, that user is made the owner of the files and the group of the files is changed to that user's login group."

So it's the same, effectively, but if someone changes the user's primary group (like 'mediamgmt') to group all services together, then the ownership isn't fucked up every time sonarr gets updated.